src/scss/includes/search_form.scss

```diff
@@ -63,7 +63,7 @@ input[type="search"] {
 // Logo
 .search_form__logo__button {
   cursor: pointer;
-  width: 36px;
+  width: 37px;
```

Comment:
This change (the other ones also) looks like a "magical" value, which reveals a very fragile layout.
We should address such questions more globally, by compositing standard margin/padding and size values, instead of fine-tuning everything constantly.

Reply:
I added this pixel because the logo seems truncated on the right (on my PC)
With the 37px width, it's fine.
![image](https://user-images.githubusercontent.com/1225909/127290025-7e0f359a-fe5a-4d1b-aab1-8cbd5477262d.png)